ftd/src/interpreter/main.rs

```diff
@@ -1349,6 +1349,85 @@ impl Document {
         Ok(None)
     }
 
+    // Returns (response, content-type, status, headers)
+    #[allow(clippy::type_complexity)]
+    pub fn get_response(
+        &self,
+    ) -> ftd::interpreter::Result<Option<(String, String, u16, fastn_resolved::Map<String>)>> {
```

Comment:
Should return `actix_web::http::StatusCode` instead of `u16`.

Reply:
We cannot do this because then we will have to add actix_web dependency in ftd crate.